src/sql/plugins/workflow/modules/userassignment/plugin/create_db_userassignment.sql

```diff
@@ -11,4 +11,15 @@ CREATE TABLE workflow_task_assign_user_information (
   id_task INT NOT NULL,
   information_key VARCHAR(255) NOT NULL,
   information_value VARCHAR(255) NULL
-);
\ No newline at end of file
+);
+
+/*==================================================================*/
+/* Table structure for table workflow_task_userassignment_assignment_notification_cf */
+/*==================================================================*/
+DROP TABLE IF EXISTS workflow_task_userassignment_assignment_notification_cf;
+CREATE TABLE workflow_task_userassignment_assignment_notification_cf (
```

Comment:
As said before, we could store a specific "From" and the id of the assignment task